src/shared/views/Mapper/Tile.js

```diff
@@ -143,8 +151,14 @@ export class _Tile extends React.Component<Props> {
         return { uri: tile.url };
     }
 
+    getOsmBuildings = () => {
```

Comment:
I would rename this `getOsmBuildingsUrl` or similar to clarify what the function returns.